examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/LoginActivity.java

```diff
@@ -0,0 +1,105 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+
+import com.google.android.gms.auth.api.Auth;
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions;
+import com.google.android.gms.auth.api.signin.GoogleSignInResult;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.api.GoogleApiClient;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private static String TAG = "LoginActivity";
+
+
+    private static final int RC_SIGN_IN = 6124;
```

Comment:
It is a random number used by Android to route the token back to the correct place. I have no clue why they require you to manager it.